run

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env sh
 set -e
 export PYTHONPATH="$(dirname $0):$PYTHONPATH"
+export PATH=$(cd `dirname $0` && pwd)/sshuttle/bin:$PATH
```

Comment:
Why not just `$(dirname $0)` like the previous line?

Reply:
I have had issues with other scripts just using `$(dirname $0)` and now use `$(cd `dirname $0` && pwd)`